drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-marshaller/pom.xml

```diff
@@ -0,0 +1,212 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.drools</groupId>
+    <artifactId>drools-wb-scenario-simulation-editor</artifactId>
+    <version>7.34.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>drools-wb-scenario-simulation-editor-kogito-marshaller</artifactId>
+  <packaging>jar</packaging>
+
+  <name>Drools Workbench - Scenario Simulation Editor - Kogito - Marshaller</name>
+  <description>Drools Workbench - Scenario Simulation Editor - Kogito - Marshaller</description>
+
+  <properties>
+    <jsinterop-annotations.version>1.0.2</jsinterop-annotations.version>
+    <jsinterop-base-version>1.0.0-beta-1</jsinterop-base-version>
+    <jsonix-scripts.version>3.0.0</jsonix-scripts.version>
+    <js.destination>${project.basedir}/src/main/resources/org/drools/workbench/scenariosimulation/kogito/marshaller/js</js.destination>
+    <gwt-jsonix-schema-compiler.version>1.1.0</gwt-jsonix-schema-compiler.version>
+    <gwt-interop-utils.version>0.3.0</gwt-interop-utils.version>
+    <maven-jaxb2-plugin.version>0.14.0</maven-jaxb2-plugin.version>
+  </properties>
```

Comment:
@manstis @yesamer is it expected that the list of dependencies differ slightly from DMN? I do not mean versions. 

https://github.com/kiegroup/kie-wb-common/blob/master/kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-marshaller/pom.xml

Reply:
@yesamer @manstis This is a good point, thank you to raise it. I'll update the properties name in order to have the same "style" present on DMN. I'll update the version as well, in case it's using an old one. But probably a change in DMN side is required for  `gwt-jsonix-schema-compiler`, in my understanding 1.1.0 is the last version generated by @gcardosi

Reply:
Ok, I reported https://issues.redhat.com/browse/DROOLS-5057 for DMN

Reply:
@jomarko

Reply:
IDK if they're required.. 

It could be they're dead baggage that can be tidied up (when @jomarko looks at the JIRA he reported).

Reply:
@manstis Thank you! @jomarko probably another "cleanup" change required in DMN is the one spotted by @danielezonca: remove `Errai.properties` and `beans.xml` files which should be useless in this module.